avocado/utils/network/interfaces.py

```diff
@@ -380,3 +380,19 @@ def restore_from_backup(self):
         else:
             raise NWException(
                 "Backup file not available, could not restore file.")
+
+    def is_available(self):
+        """Check if interface is available.
+
+        This method checks if the interface is available.
+
+        rtype: bool
+        """
+        cmd = 'ip link show dev {}'.format(self.name)
+        try:
+            run_command(cmd, self.host, sudo=True)
```

Comment:
afaict, you don't need sudo to run a `ip l show`, right?

Reply:
Correct, I've removed it.